grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -339,6 +339,10 @@ class Location implements Comparable<Location>, java.io.Serializable {
         return active ? LocationStatus.ENABLED : LocationStatus.DISABLED
     }
 
+    Boolean isApprovalRequired() {
+        return supportedActivities?.contains(ActivityCode.APPROVE_REQUEST.id)
+    }
+
```

Comment:
I can see that there is a method called `supports` maybe it would be better to use it?
I mean:
```
Boolean isApprovalRequired() {
        return supports(ActivityCode.APPROVE_REQUEST.id)
}
```

Reply:
Yeah, thanks for pointing that out, I forgot to ask about it as well - if we should use contains like I did, or supports, because supports also checks for locationType’s supportedActivities and I was not sure whether the request approval is supposed to work if we add this activity to a locationType

Reply:
I think it should use `supports`

Reply:
Agreed